.rubocop.yml

```diff
@@ -0,0 +1,41 @@
+AllCops:
```

Comment:
I suggest you use the .rubocop.yml` file for ROR istead of simply for Ruby. https://github.com/microverseinc/linters-config/blob/master/ror/.rubocop.yml (Same for stickler)